source/_includes/code_examples/code_example.liquid

```diff
@@ -1,3 +1,11 @@
+<div class="sl-c-to-playground">
+{%- if code.scss.size == 1 -%}
+  <a href="/playground#{{ playgroundLinks.scss }}" class="sl-c-to-playground__link" target="_blank" data-syntax="scss"><span class="sl-c-to-playground__highlight">Edit on</span> Playground</a>
+{%- endif -%}
+{%- if code.sass.size == 1 -%}
+  <a href="/playground#{{ playgroundLinks.sass }}" class="sl-c-to-playground__link" target="_blank" data-syntax="sass"><span class="sl-c-to-playground__highlight">Edit on</span> Playground</a>
+{%- endif -%}
+</div>
```

Comment:
@jamesnw I wrapped this in a div because if I didn't, it automatically added a paragraph tag. I would prefer to not have a wrapper here I think, is that doable/easy to achieve?

Reply:
As best I can tell, the template has to start with a block element. That gives us a few options-

- Wrap the whole file in a <div>
- Start the file with `<div></div>`

The first seems less surprising, but either could work, depending on your needs

Reply:
@jamesnw I think I'll go with wrapping everything in a div. I think it gets closer to the design like this too, stay tuned.